### ISSUE TITLE
Update PHP version on the server to 8.0

### ISSUE THREAD

**Issue by ob-ivan** [ob-ivan/ommacomics#36]
The currently used PHP version is 7.2, which has met its end of life in 2020.

Consider upgrading to PHP 8.

Some related reading: https://stackoverflow.com/questions/62870093/how-to-update-to-php-7-4-from-php-7-x-on-centos-7

**Comment by ob-ivan** [comment]
Might be done by #62.

**Comment by ob-ivan** [comment]
Tutorial: https://www.microhost.com/docs/tutorial/how-to-install-php-8-in-centos-7/

Migration plan:
- [ ] # yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
- [ ] # yum install https://rpms.remirepo.net/enterprise/remi-release-7.rpm
- [ ] # yum install -y --enablerepo=remi-php80 php php-cli
- [ ] # php -v
- [ ] # yum install -y --enablerepo=remi-php80 php-iconv php-json php-mysqli php-zip

Extensions:
- iconv
- json
- mysqli
- zip

**Comment by ob-ivan** [comment]
Update successful!

**Comment by ob-ivan** [comment]
Reading a chapter results in a following error:

```
Compile Error: Declaration of App\Kernel::configureRoutes(Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator $routes) must be compatible with Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait::configureRoutes(Symfony\Component\Routing\RouteCollectionBuilder $routes)
```

**Comment by ob-ivan** [comment]
The above error goes away after upgrading Symfony to version 5.3.

Resolution of this issue depends on how #28 goes.

**Comment by ob-ivan** [comment]
Closed via #28.
